bundles/org.openhab.binding.sonos/README.md

```diff
@@ -58,7 +58,7 @@ The devices support the following channels:
 | batterycharging      | Switch    | R           | Indicator set to ON when the battery is charging                                                                                                          | Move, Roam, RoamSL                                              |
 | batterylevel         | Number    | R           | Current battery level                                                                                                                                     | Move, Roam, RoamSL                                              |
 | clearqueue           | Switch    | W           | Suppress all songs from the current queue                                                                                                                 | all                                                             |
-| codec                | String    | R           | Name of codec currently being decoded                                                                                                                     | Arc, Arc SL, PLAYBAR, PLAYBASE, Beam, Amp                       |
+| codec                | String    | R           | Name of codec currently being decoded                                                                                                                     | Arc, ArcSL, PLAYBAR, PLAYBASE, Beam, Amp                        |
```

Comment:
If the list refers to actual product names, I believe it was correct before:
```suggestion
| codec                | String    | R           | Name of codec currently being decoded                                                                                                                     | Arc, Arc SL, PLAYBAR, PLAYBASE, Beam, Amp                       |
```

See https://support.sonos.com/en-us/products/arc-sl

Reply:
It refers to thing types, look at column title, that is the reason why I changed ArcSL.